pkg/action/executor/binder.go

```diff
@@ -0,0 +1,21 @@
+package executor
+
+import (
+
+	"k8s.io/kubernetes/pkg/api"
+	client "k8s.io/kubernetes/pkg/client/unversioned"
+
+	"github.com/golang/glog"
+)
+
+// Binder is used to schedule pod onto node.
+type binder struct {
+	*client.Client
+}
+
+// Bind just does a POST binding RPC.
+func (b *binder) Bind(binding *api.Binding) error {
+	glog.V(2).Infof("Attempting to bind %v to %v", binding.Name, binding.Target.Name)
```

Comment:
What happened if Bind failed for some reasons? Do we want to retry here or k8s will retry internally?